src/char/sorceress/light_sorc.py

```diff
@@ -70,52 +70,92 @@ def kill_shenk(self) -> bool:
-            self._chain_lightning(cast_pos_abs, spray=60)
-        self._cast_static()
-        # move a bit back
-        pos_m = self._screen.convert_abs_to_monitor((100, 0))
+        wait(0.1)
+        pos_m = self._screen.convert_abs_to_monitor((-550, 230))
+        self.pre_move()
+        self.move(pos_m, force_move=True)
+        wait(0.1)
+        self._pather.offset_node(226, (-80, 60))
+        self._pather.traverse_nodes([226], self, time_out=1.0, force_tp=True)
+        self._pather.offset_node(226, (80, -60))
+        wait(0.5)
+        self._frozen_orb((-150, -130), spray=10)
+        self._chain_lightning((200, -185), spray=20)
+        self._chain_lightning((-170, -150), spray=20)
+        wait(0.5)
+        self._pather.traverse_nodes_fixed([(1110, 15)], self)
+        self._pather.traverse_nodes([300], self, time_out=1.0, force_tp=True)
+        pos_m = self._screen.convert_abs_to_monitor((300, 150))
+        self.pre_move()
+        self.move(pos_m, force_move=True)
+        wait(0.1)
+        self._frozen_orb((-170, -100), spray=40)
+        self._chain_lightning((-300, -100), spray=10)
+        self._chain_lightning((-300, -90), spray=10)
+        self._lightning((-300, -110), spray=10)
+        wait(0.5)
+        # Move back outside and attack
+        pos_m = self._screen.convert_abs_to_monitor((-430, 230))
+        self.pre_move()
+        self.move(pos_m, force_move=True)
+        self._pather.offset_node(304, (0, -80))
+        self._pather.traverse_nodes([304], self, time_out=1.0, force_tp=True)
+        self._pather.offset_node(304, (0, 80))
+        wait(0.1)
+        self._frozen_orb((175, -170), spray=40)
+        self._chain_lightning((-170, -150), spray=20)
+        self._chain_lightning((300, -200), spray=20)
+        self._chain_lightning((-170, -150), spray=20)
+        wait(0.5)
+        # Move back inside and attack
+        pos_m = self._screen.convert_abs_to_monitor((350, -350))
+        self.pre_move()
+        self.move(pos_m, force_move=True)
+        pos_m = self._screen.convert_abs_to_monitor((100, -30))
+        self.pre_move()
+        self.move(pos_m, force_move=True)
+        # Attack sequence center
+        self._frozen_orb((0, 20), spray=40)
+        self._lightning((-50, 50), spray=30)
+        self._lightning((50, 50), spray=30)
+        wait(1.0)
+        # Move inside
+        pos_m = self._screen.convert_abs_to_monitor((40, -30))
         self.pre_move()
         self.move(pos_m, force_move=True)
-        cast_pos_abs = np.array([-36, -126])
-        for _ in range(atk_len_trav_2):
-            self._chain_lightning(cast_pos_abs, spray=60)
-            self._lightning(cast_pos_abs, spray=60)
+        # Attack sequence to center
+        self._chain_lightning((-150, 100), spray=20)
+        self._chain_lightning((150, 200), spray=40)
+        self._chain_lightning((-150, 50), spray=40)
+        # Move outside since the trav.py expects to start searching for items there if char can teleport
```

Comment:
Please actually move outside. In the videos you sent me you can see that sometimes potential loot is not seen because it is outside of view or hud is hiding it.